### DISCUSSION TITLE
打扰了，请问能添加上传功能吗

### DISCUSSION THREAD

**Discussion by chiron688** [AlistGo/alist#207] [Ideas]
如题，希望能在alist中一键上传自己的文件，不需要再额外打开阿里云盘等的网站进行上传

**Comment by chiron688** [comment]
还有，不明白docker版本如何更新，stop docker之后进行pull最新代码，然后再重启，版本号还是旧版本号

**Comment by xhofe** [comment]
web端暂时不会添加上传的功能，上传可以使用WebDAV，能支持的会尽量添加，但是有的上传实在太复杂了就无能为力了。
docker更新需要删除容器，参考：[备用](https://store.heytapimage.com/cdo-portal/feedback/202112/12/1e64956f796e95070bed2d7f2c61bdd5.png)
![image](https://user-images.githubusercontent.com/36558727/145702676-f359d8a2-2ffa-4919-8f30-42e7fad80826.png)

**Comment by ghost** [comment]
阿里云盘支持webdav上传么现在？

**Reply by xhofe** [reply]
https://alist-doc.nn.ci/docs/intro#webdav-%E6%94%AF%E6%8C%81

**Reply by lyj0309** [reply]
> https://alist-doc.nn.ci/docs/intro#webdav-%E6%94%AF%E6%8C%81

大佬，可以参考下这个，支持网页上传的
https://github.com/cloudreve/Cloudreve
